package.json

```diff
@@ -35,7 +35,7 @@
     "react-calendly": "^4.1.1",
     "react-dom": "18.2.0",
     "react-icons": "^4.7.1",
-    "rudder-sdk-js": "^2.25.0",
+    "rudder-sdk-js": "^2.34.0",
```

Comment:
Any specific reason for updating the RudderStack version? Any have you tested if everything works as expected even with the updated version of RudderStack?

Reply:
```suggestion
    "rudder-sdk-js": "^2.25.0",
```